lms/djangoapps/courseware/views/views.py

```diff
@@ -103,6 +103,7 @@
 from xmodule.modulestore.exceptions import ItemNotFoundError, NoPathToItem
 from xmodule.tabs import CourseTabList
 from xmodule.x_module import STUDENT_VIEW
+from learner_dashboard.programs import get_course_ids
```

Comment:
This import is not sorted.

Reply:
@dyudyunov does not get your point. It is in proper order.

Reply:
Not get anyway. What order? alphabet? It is not necessary :)